min.py

```diff
@@ -146,9 +177,29 @@ def get_min_scipy_version(min_python, min_numpy):
         MIN_PYTHON = get_min_python_version()
     MIN_NUMBA, MIN_NUMPY = get_min_numba_numpy_version(MIN_PYTHON)
     MIN_SCIPY = get_min_scipy_version(MIN_PYTHON, MIN_NUMPY)
+
     print(
         f"python: {MIN_PYTHON}\n"
         f"numba: {MIN_NUMBA}\n"
         f"numpy: {MIN_NUMPY}\n"
         f"scipy: {MIN_SCIPY}"
     )
+
+    pkgs = {
+        "numpy": MIN_NUMPY,
+        "scipy": MIN_SCIPY,
+        "numba": MIN_NUMBA,
+        "python": MIN_PYTHON,
+        "python-version": MIN_PYTHON,
+    }
+
+    fnames = [
+        "pyproject.toml",
+        "requirements.txt",
+        "environment.yml",
+        ".github/workflows/github-actions.yml",
+        "README.rst",
+    ]
+
+    for pkg_name, pkg_version in pkgs.items():
+        print(find_pkg_mismatches(pkg_name, pkg_version, fnames))
```

Comment:
Based on my change suggestion above, I think this will become:

```
for name, version, fname, line_num in find_pkg_mismatches(pkg_name, pkg_version, fnames):
    print(f'Package Mismatch Found: "{pkg_name}" "{version}" in {fname}:{line_num}\n'
```